main.go

```diff
@@ -393,6 +401,9 @@ func (b *Backend) updateCallStats(t shortTraceMsg) {
 	b.Stats.MinLatency = time.Duration(int64(math.Min(float64(b.Stats.MinLatency), float64(t.CallStats.Latency))))
 	b.Stats.Rx += int64(t.CallStats.Rx)
 	b.Stats.Tx += int64(t.CallStats.Tx)
+	// automatically update the global stats
+	// Read/Write Lock is not required here
+	globalConnStatsRWMutex.RLock()
```

Comment:
```suggestion
	globalConnStatsRWMutex.Lock()
	conns := globalConnStats
	globalConnStatsRWMutex.Unlock()
```

Change for loop to use `conns` below.

Wow this is clumsy. Mutex on 'b.Stats', lookup on all globalConns.